### PR TITLE
adding data.frame method for drop_units

### PR DESCRIPTION
**Pull request by kendonB**
addresses https://github.com/r-quantities/units/issues/187

### CODE REVIEW

File: R/make_units.R [hunk 1]
```diff
@@ -440,3 +440,12 @@ drop_units.units <- function(x) {
   x
 }
 
+#' @export
+drop_units.data.frame <- function(.data){
```
Comment (Enchufa2) [inline R/make_units.R]: An S3 method should be consistent with its generic: in this case, the argument must be `x` instead of `.data`.

**Comment by Enchufa2** [comment]
Also, not required, but it would be nice if you could provide a small test for this new functionality and/or update the NEWS.

**Comment by Enchufa2** [comment]
@edzer Please, have a look and merge this if you are ok with the addition.

**Comment by Enchufa2** [comment]
@edzer Kind reminder. ;-)

**Comment by edzer** [comment]
Thanks for reminding!

**Pull request merged**